lcc_lokavaluto_app_connection/models/account_invoice.py

```diff
@@ -0,0 +1,32 @@
+from odoo import models, fields, api
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    has_numeric_lcc_products = fields.Boolean(
+        string="Has lcc numeric products",
+        compute="_compute_has_numeric_lcc_products",
+        store=True,
+    )
+
+    @api.one
+    @api.depends("state", "invoice_line_ids.product_id", "invoice_line_ids.price_total")
```

Comment:
not sure what depends do... so not sure why depend on `price_total` ?

Reply:
Hum copy/paste I think